Simple Login Screen/LoginViewController.swift

```diff
@@ -30,14 +30,112 @@ class LoginViewController: UIViewController {
 
     
     @IBAction func signupButtonPressed(_ sender: Any) {
+        
+        //Clear keyboard
```

Comment:
Sign up button is pressed and you save the data to the arrays but we must make sure the data is good before we save it. there should be some check here to ensure that each field is good to go